src/foundry/common/data/data.mjs/tokenData.d.ts

```diff
@@ -554,10 +486,16 @@ export class TokenData extends DocumentData<
   static defineSchema(): TokenDataSchema;
 
   /**
-   * The default icon used for newly created Item documents
+   * The default icon used for newly created Token documents
    * @defaultValue `CONST.DEFAULT_TOKEN`
    */
   static DEFAULT_ICON: string;
+
+  /** @override */
```

Comment:
```suggestion
  /**
   * Migrate TokenData attributes to the new inner LightData structure.
   * This can be safely removed after several major versions have passed. Maybe V12?
   * @override
   */
```